Weekly Changelog 2026-05-18 to 2026-05-25 (#87216)
CleverRaven/Cataclysm-DDA: data/changelog.txt

Vehicle turrets and tools draw from multimag pockets
Recipes can mark steps as unattended; player can wait, walk away, or set a timer
Getting an Exodii Cyberframe locks you out of Hub 01 content
CBMs can define replaced bodyparts, and removing a bionic limb removes CBMs in that limb

> Exodii cyborgs can install and uninstall bionic limbs at autodocs/etc
> Exodii surgeon can sell, install, uninstall, and repair bionic limbs
> Bionic limbs cannot be mended with splints, but can be repaired from 0
> More nuanced Hub 01-Exodii full cyborg aggression
> Charged tools can be used on unattended crafting steps
> Unattended liquid crafts wait for you to pour them

Reviewer: ```suggestion
CBMs can define replaced bodyparts, and removing a bionic limb removes CBMs in that limb
Exodii can install and uninstall bionic limbs 
Bionic limbs cannot be mended with splints, but can be repaired from 0
More nuanced Hub 01-Exodii full cyborg aggression
Charged tools can be used on unattended crafting steps
Unattended liquid crafts wait for you to pour them
```

Overgrown labyrinths
Labyrinth specific CBMs:  anchor; laser reader; psyshield; artificial hair; blood biofactory; antigravity jump
The Exodii can turn you into a full-blown cyborg
Introduction of three new small mammals
Styrofoam
10 more rooms for labyrinth level 1
If the Exodii trust you enough they'll give you a Cyberframe
Add new vegetables
Cash register impulse buy item group
Adding High Density Polyethylene
Added fire extinguishers to many locations

> Add new vegetables

Reviewer: ```suggestion
If the Exodii trust you enough they'll give you a Cyberframe
```

Update to ImGui 1.91.9b
Move appliance infobox to imgui
Display some items by volume, weight or length instead of count
Add bulk reading map caches to electronic device menu
Updated Tainted Bone description to reflect it is not kilnable
Show monster difficulty rating in the look panel
Update vehicle shape menu

> Supernatural Power menu says `Choose a Supernatural Power` 

Reviewer: ```suggestion
```

[BPerks/MM/Sorc] XP mods can level up more than once at a time
Aftershock: Add industrial habitat building.
Add "XedraWood: Stone and Sorcery" mod to the CDDA repository
[XE] Add the vampire hunter faction and its base
XE: Horse Familiar for Dhampir
[XedraWood] Add the hollow villages to the mod
[XE] Edit the living-drinker vampire weakness
Aftershock: Slightly nerf Veles robots.
[MoM: Psychic Scream] Make the Psychic trait visible
[Sky Island] Keep exit room mission markers
[MoM] Dimensional Anchors will eventually cancel your concentration powers
[MoM] Using censer from the Highlands raises your Nether Attunement
[XE] Two new blood art refinements
Aftershock: Railguns use power cartridges instead of requiring an UPS
Aftershock: Add light switches to the vanishing arc.

> `Blank body` can copy your appearance

Reviewer: ```suggestion
```

Nobody wants to cooperate with psychopaths
Zombie corpses aren't useful
Stepped many many weapon recipes
Fewer enemies roaming the halls of level 1 labyrinth
now eggplant seed is flagged and inserted in the farming seed item group
Joint Torsion Ratchet CBM doesn't allow for perpetual motion cyborgs
Mail is too strong. Plate, too weak
Nerf Hub 01 bionic limb performance, buff protection

> Psychopaths cannot continue as a follower on death.

Reviewer: ```suggestion
Joint Torsion Ratchet CBM doesn't allow for perpetual motion cyborgs
```

Stop spamming player log when NPCs are hot
Stop ref center guard EOCs from firing in alternate dimensions
Uncomfortable limbs must exist
Fix conflicting keybind in electronic device menu
[Xedra Evolved] Fix Verdant Scourge conditions
ASCII: Fix idle detection in asciinema
Spawn military weapons at the Return to Sender convoy ambush site
Fix multimag gun info crash and per-pocket capacity misreads
Stop game from reacting to clicks aimed at ImGui windows
Spawn military weapons at the Return to Sender convoy ambush site
Descriptions of the items smoked meat, smoked fish, smoked lobster and smoked mollusk
Fix crash when opening inventory with bare barrel-swappable gun
Clarified that each cake, pie, and pizza item is just a slice, and not the whole thing.
multi-activity fetch segfault fix
Fix crash spawning items with a null default ammo or magazine slot
Minimap no longer draws over the Mouse View UI
Remove the LARGE flag from EXODII_CYBERFRAME_GENERIC
Fix SDL3 sound freeze when many sound effects play at once
Recalculate enchantment cache after loading bionics in Character::load()
Items that are decharged aren't strew out everywhere
Jumbo Treefrog Description

> Scarves cover beaks and muzzles

Reviewer: ```suggestion
```
